src/main/java/frc/robot/commands/ShooterPressed.java

```diff
@@ -50,7 +50,7 @@ public void execute() {
       // Otherwise, alignment is checked.
       if (i > 0 || !LLEnabled || m_LimelightSubsystem.calculatePID() == 0.0) {
         if (i == 0) {
-          m_CDSSubsystem.CDSBeltToggle(false);
+          m_CDSSubsystem.CDSBeltToggle(false, 0.95);
```

Comment:
Constants for magic numbers

Reply:
You created a constant: CDSAutoBeltSpeed but you aren't using it, yet.